examples/bigquery-snowflake-tables-migration-utility/DDL_Extractor/Snowflake DDL Extraction.md

```diff
@@ -0,0 +1,21 @@
+The Snowflake DDL Extraction Script does the following functionalities:
+
+The script connects to Snowflake Database through the snowflake-python connector 
+The script uses the snowflake get_ddl command to retieve the table schema information
+The script produces the "create table" statement using the schema information and store the extracted ddl in the specified gcs path
+The status of each table conversion is logged in the audit table in the target datset.
+Below packages are need to run the script: google-cloud-secret-manager google-cloud-bigquery google-cloud-storage google-api-core snowflake-connector-python 
+
+Steps to run this script:
+
+Create the sf-ddl-extraction-config.json file and place it in the gcs bucket.
+
+Create the object_name_mapping.json file and place it in the gcs bucket.
```

Comment:
Could you please add more detail on how this file is should be created, what's it purpose, what do it's fields mean ... ?